variables.tf

```
@@ -99,6 +99,12 @@ variable "os_disk_storage_account_type" {
   default     = "StandardSSD_LRS"
 }
 
+variable "os_disk_size_gb" {
+  description = "The size of this OS disk."
```

We could clarify unit in description:

```suggestion
  description = "The size of this OS disk in gigabytes (GB)."
```

---

Good catch